pkg/security/ImageScanService.go

```diff
@@ -654,7 +583,25 @@ func (impl ImageScanServiceImpl) VulnerabilityExposure(request *security.Vulnera
 	return vulnerabilityExposureListingResponse, nil
 }
 
-func (impl ImageScanServiceImpl) GetArtifactVulnerabilityStatus(ctx context.Context, request *bean2.VulnerabilityCheckRequest) (bool, error) {
+func (impl *ImageScanServiceImpl) CalculateSeverityCountInfo(vulnerabilities []*bean3.Vulnerabilities) *bean3.SeverityCount {
+	diff := bean3.SeverityCount{}
+	for _, vulnerability := range vulnerabilities {
+		if vulnerability.IsCritical() {
+			diff.Critical += 1
+		} else if vulnerability.IsHigh() {
+			diff.High += 1
+		} else if vulnerability.IsMedium() {
+			diff.Medium += 1
+		} else if vulnerability.IsLow() {
+			diff.Low += 1
+		} else if vulnerability.IsUnknown() {
+			diff.Unknown += 1
+		}
```

Comment:
this could have been part of method belongs to `bean3.SeverityCount`

Reply:
closing as we  discussed